From: kernel test robot <lkp@intel.com>
Subject: mm/secretmem: secretmem_iops can be static

mm/secretmem.c:173:31: warning: symbol 'secretmem_iops' was not declared. Should it be static?

Link: https://lkml.kernel.org/r/YkXc0RjJYQoGZZ8A@57de4820519d
Fixes: 24209f2f4d62 ("mm/secretmem: fix panic when growing a memfd_secret")
Signed-off-by: kernel test robot <lkp@intel.com>
Reported-by: kernel test robot <lkp@intel.com>
Cc: Axel Rasmussen <axelrasmussen@google.com>
Cc: Matthew Wilcox <willy@infradead.org>
Cc: Mike Rapoport <rppt@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/secretmem.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/secretmem.c~mm-secretmem-fix-panic-when-growing-a-memfd_secret-fix
+++ a/mm/secretmem.c
@@ -170,7 +170,7 @@ static int secretmem_setattr(struct user
 	return simple_setattr(mnt_userns, dentry, iattr);
 }
 
-const struct inode_operations secretmem_iops = {
+static const struct inode_operations secretmem_iops = {
 	.setattr = secretmem_setattr,
 };
 
_
